app/views/users/_register_form.html.erb

```diff
@@ -0,0 +1,10 @@
+
+<%= form_for @user, url: sessions_path do |f| %>
+
+    <%= f.label :username %>
+    <%= f.text_field :username %>
+    <%= f.label :password %>
+    <%= f.password_field :password %>
+    <%= f.submit "Log In" %>
+
+<% end %>
```

Comment:
this looks very similar to the login partial, can we roll it up into one with different actions?  also, I think the action might be wrong.